index.js

```diff
@@ -132,6 +132,7 @@ LiveServer.start = function(options) {
 	var staticServerHandler = staticServer(root);
 	var wait = options.wait || 0;
 	var browser = options.browser || null;
+	var authPath = options.authPath || null;
```

Comment:
I think we should use htpasswd (or authPath, but I think I prefer the former) for both the cli flag and the api param for consistency reasons (I'm aware that not all current params actually are consistent :disappointed:).